server/db/migrations/20200124113424-create-user.js

```diff
@@ -0,0 +1,40 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      type: Sequelize.INTEGER,
+    },
+    memberID: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      primaryKey: true,
+    },
+    fullName: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    password: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    isAdmin: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
```

Comment:
@hakoemmy 

thanks, bro 
@AbdoulNuru  will take care of it tomorrow 
before he merges we will all need to sit and review it 
database setup is important  (done well it can save us a lot of time)

Reply:
@hakoemmy @blaise82 I have change the isAdmin datatype to BOOLEAN as requested.